bats/tests/k8s/wordpress.bats

```diff
@@ -60,50 +43,12 @@ verify_wordpress() {
     assert_output --regexp "(Just another WordPress site|<title>User&#039;s Blog!</title>)"
 }
 
-@test 'verify nginx before upgrade' {
-    verify_nginx
-}
-
-@test 'verify wordpress before upgrade' {
+@test 'verify wordpress was deployed' {
```

Comment:
Since `verify_wordpress` is called only once, it should be inlined into the test.